responses/sv/HassTimerStatus.yaml

```diff
@@ -0,0 +1,94 @@
+---
+language: sv
+responses:
+  intents:
+    HassTimerStatus:
+      default: |
+        {% set num_timers = slots.timers | length %}
+        {% set active_timers = slots.timers | selectattr('is_active') | list %}
+        {% set num_active_timers = active_timers | length %}
+        {% set paused_timers = slots.timers | rejectattr('is_active') | list %}
+        {% set num_paused_timers = paused_timers | length %}
+        {% set next_timer = None %}
+
+        {% if num_timers == 0: %}
+          Det finns inga timers.
+        {% elif num_active_timers == 0: %}
+          {# No active timers #}
+          {% if num_paused_timers == 1: %}
+            {% set next_timer = paused_timers[0] %}
+            Timern är pausad. 
+          {% else: %}
+            {{ num_paused_timers }} pausade timers.
+          {% endif %}
+        {% else: %}
+          {# At least one active timer #}
+          {% if num_active_timers == 1: %}
+            {% set next_timer = active_timers[0] %}
+          {% else: %}
+            {# Get active timer that will finish soonest #}
+            {% set sorted_timers = active_timers | sort(attribute='total_seconds_left') %}
+            {% set next_timer = sorted_timers[0] %}
+            {{ num_active_timers }} pågående timers.
+          {% endif %}
+
+          {% if num_paused_timers == 1: %}
+            1 pausad timer.
+          {% elif num_paused_timers > 0: %}
+            {{ num_paused_timers }} pausade timers.
+          {% endif %}
+        {% endif %}
+
+        {% if next_timer: %}
+          {# At least one active timer #}
+          {% if (next_timer.rounded_hours_left == 1) and (next_timer.rounded_minutes_left > 1): %}
+            1 timme och {{ next_timer.rounded_minutes_left }} minuter
+          {% elif (next_timer.rounded_hours_left ==1) and (next_timer.rounded_minutes_left == 1): %} 
+            1 timme och 1 minut
+          {% elif (next_timer.rounded_hours_left > 0): %}
```

Comment:
This will catch next_timmer.rounded_hours_left == 2 and say "1 timme" 

Since it match it will not go to the next else-if to see if the hours is more the 1 hour.

Please add a test case for this case also.

Reply:
You're right, I'll fix it. Is there a good way to add fixtures for specific tests? Because if I add a new timer to the fixtures file it affects some other tests (du har ~~två~~ tre pausade timers)

Reply:
Not sure, will check with the other language_leaders to see if there is any input on more dynamic test cases. It should have a testcase where no times exists also so we need someway to define it in the testcase.

Reply:
I've fixed it and added a test case for it. Should I open this for review in the meantime until we figure out how we can do more specific tests?

Reply:
Sure, we can fix and add more testcase if we get a better solution.